frontend/src/interfaces/featureToggle.ts

```diff
@@ -96,6 +97,7 @@ export interface IFeatureEnvironment {
     lastSeenAt?: string;
     yes?: number;
     no?: number;
+    releasePlans?: IReleasePlan[];
```

Comment:
you may need to add the new fields I added to this type if you want started date and transition condition (probably next PR)